go get fails with error

**Describe the bug**

Installing swag recursively results in an install error.

**To Reproduce**

```
go get -u github.com/swaggo/swag/...
```

Results in:

```
# github.com/swaggo/swag/example/basic/api
example/basic/api/api.go:21:4: no new variables on left side of :=
```

**Your swag version**
e.g. 1.6.1

**Your go version**
e.g. 1.11.5

--- a/example/basic/api/api.go
+++ b/example/basic/api/api.go
@@ -18,8 +18,13 @@ import (
 // @Failure 404 {object} web.APIError "Can not find ID"
 // @Router /testapi/get-string-by-int/{some_id} [get]
 func GetStringByInt(c *gin.Context) {
-	_ := web.Pet{}
-	//write your code
+	var pet web.Pet
+	if err := c.ShouldBindJSON(&pet); err != nil {
+		// write your code
+		return
+	}
+
+	// write your code
 }
 
 // GetStructArrayByString example
@@ -35,7 +40,7 @@ func GetStringByInt(c *gin.Context) {
 // @Failure 404 {object} web.APIError "Can not find ID"
 // @Router /testapi/get-struct-array-by-string/{some_id} [get]
 func GetStructArrayByString(c *gin.Context) {
-	//write your code
+	// write your code
 }
 
 // Upload example
@@ -50,7 +55,7 @@ func GetStructArrayByString(c *gin.Context) {
 // @Failure 404 {object} web.APIError "Can not find ID"
 // @Router /file/upload [post]
 func Upload(ctx *gin.Context) {
-	//write your code
+	// write your code
 }
 
 // AnonymousField example

